docs/codeql/codeql-workshops/python/codeql-dataflow-sql-injection/queries.xml

```diff
@@ -0,0 +1 @@
+<queries language="python"/>
```

Comment:
We should never need these when we have `qlpack.yml` files -- go ahead and remove them.

Reply:
done